src/main.rs

```diff
@@ -27,15 +27,18 @@ use std::{
     string::ToString,
 };
 
-use camino::{Utf8Path, Utf8PathBuf};
-use clap::Parser;
-use cmd::{AllArgs, Cli, SubCommand};
+// External dependencies:
+use camino::{Utf8Path, Utf8PathBuf};  // UFT8 Paths
+use clap::Parser;  // Command-Line Interface Framework
+use cmd::{AllArgs, Cli, SubCommand};  // Running shell commands
+use tempdir::TempDir;  // Nomen est omen
```

Comment:
For us not so rust-savy, it's nice to see at a glace, which package does what, because to the uninitiated "clap" sounds like a stripper term.